samples/chat-app/server/FSharp.Data.GraphQL.Samples.ChatApp.fsproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <RootNamespace>FSharp.Data.GraphQL.Samples.ChatApp</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="FsToolkit.ErrorHandling.TaskResult" />
+    <PackageReference Include="GraphQL.Server.Ui.Playground" />
+    <PackageReference Include="GraphQL.Server.Ui.Voyager" />
+    <PackageReference Include="HotChocolate.AspNetCore" />
```

Comment:
@xperiandri are we sure we want to reference our "competitor" HotChocolate? 🙈 
I know it's because of the handy playground, voyager and Banana Cake Pop things, but I'm not sure it's worth it...
In my view, FSharp.Data.GraphQL should not, by principle, reference HotChocolate in any way. And I know it's in the sample only, but still.
As for me, just as an example, I've been using [Postman](https://learning.postman.com/docs/sending-requests/graphql/graphql-overview/) to build my own request collections.

Reply:
I just like their UI. Can we use that UI ourself?

Reply:
As I said: I'm not in favor of having any NuGet package reference to HotChocolate in FSharp.Data.GraphQL's code (be it a sample or not). Of course people can add it to their projects that use FSharp.Data.GraphQL, just like you did here.